packaging/arch/PKGBUILD

```diff
@@ -36,23 +42,35 @@ prepare() {
   # above describes.
   mkdir -p "$(dirname "$GOPATH/src/${_gourl}")"
   ln --no-target-directory -fs "$srcdir/$pkgname" "$GOPATH/src/${_gourl}"
+  # Patch snap-seccomp build flags not to link libseccomp statically.
+  sed -i -e 's/-Wl,-Bstatic -lseccomp -Wl,-Bdynamic/-lseccomp/' "$srcdir/$pkgname/cmd/snap-seccomp/main.go"
+  # Apply some upstream fixes not present in the release.
```

Comment:
There's just one left, I was really hoping to just merge them all and get rid of this. The only kind of patches that we want to carry here (in snapd master, not in the actual packaging in arch git repos) is the one that doesn't need to carry patches since we can just apply those.